src/building/dock.h

```diff
@@ -3,6 +3,10 @@
 
 #include "building/building.h"
 #include "map/point.h"
+#include "game/resource.h"
+#include "figure/figure.h"
```

Comment:
The previous two includes are only needed on the C file, so please move them there.